docs/tools/member.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+import os
+import argparse
+import yaml
+
+from github import Github
+
+
+def main(access_token, people_yml):
+    '''update the Pangeo organization with any new members'''
+    # authentication with GitHub
+    g = Github(access_token)
+    org = g.get_organization('pangeo-data')
+
+    # people.yml is in the pangeo docs
+    with open(people_yml) as f:
+        people = yaml.safe_load(f)
+
+    github_ids = [p['github'] for p in people]
+
+    # pangeo-members = 2180253
+    team = org.get_team(2180253)
+
+    # get existing GitHub members
+    existing_org_logins = list(member.login for member in org.get_members())
+    existing_team_logins = list(member.login for member in team.get_members())
+
+    # add those members not yet in the GitHub org
+    for member in github_ids:
+        user = g.get_user(member)
```

Comment:
do we need to catch an error here if `member` is not actually a valid github user?